en/docs/develop-components/develop-components-with-git.md

```diff
@@ -65,6 +65,12 @@ For example, when you [work with the Micro Integrator (MI) runtime in Choreo](./
             !!! info "Note"
                 Choreo currently does not support accessing private repositories in other organizations.
 
+#### Automatically Pull Latest Submodule Versions
+Choreo allows you to always pull the latest versions of your Git submodules from their respective repositories. You can enable this feature by toggling Pull Latest Submodules under Build Configurations on the Build page. Currently, this feature is only supported for Micro Integrator (MI) components.
```

Comment:
With regard to enabling the feature, let's follow the Google style guide and use the proper terminology. 
Also, the statement "Currently, this feature is only supported for Micro Integrator (MI) components." doesn't seem correct because we don't have "Micro Integrator (MI) components". Need to specifically mention that it is supported for the WSO2 MI buildpack and mention the exact component types.